jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -209,7 +209,7 @@ def _calculate_statistics(self, *args, **kwargs):
             # try extracting from the region
             continuum_mask = np.array([0, len(spectral_axis)-1])
             mark_x = {'left': np.array([]),
-                      'center': np.array([spectral_axis.value[0], spectral_axis.value[-1]]),
+                      'center': np.array([min(spectral_axis.value), max(spectral_axis.value)]),
```

Comment:
Does this work as expected when the unit is in Hz? I can't keep track what is what, but at one point I remember @rosteen had to fix a bug upstream to accept reversed axis order or something.

Reply:
I just tested after converting to Hz and it appears to work.